mindsdb/interfaces/skills/skills_controller.py

```diff
@@ -0,0 +1,156 @@
+from typing import Dict, List
+
+from mindsdb.interfaces.storage import db
+from mindsdb.interfaces.database.projects import ProjectController
+
+
+class SkillsController:
+    '''Handles CRUD operations at the database level for Skills'''
+
+    def __init__(self, project_controller: ProjectController = None):
+        if project_controller is None:
+            project_controller = ProjectController()
+        self.project_controller = project_controller
+
+    def get_skill(self, skill_name: str, project_name: str = 'mindsdb') -> db.Skills:
+        '''
+        Gets a skill by name. Skills are expected to have unique names.
+
+        Parameters:
+            skill_name (str): The name of the skill
+            project_name (str): The name of the containing project
+
+        Returns:
+            skill (db.Skills): The database skill object
+
+        Raises:
+            ValueError: If `project_name` does not exist
+        '''
+
+        project = self.project_controller.get(name=project_name)
+        return db.Skills.query.filter(
```

Comment:
Do we have a unique constraint on `skill name` + `project id`?
I think we are using this combination as a surrogate PK here.